wolfcrypt/src/asn.c

```diff
@@ -379,7 +364,7 @@ struct tm* mqx_gmtime(const time_t* clock, struct tm* tmpTime)
 
 time_t ksdk_time(time_t* timer)
 {
-    time_t localTime;
+    static time_t localTime;
```

Comment:
Ah I see it now. Okay we are good. Will revert this one and investigate the other questions about time_t. Thanks Todd